<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="BlueprintView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionPen"/>
    <addaction name="separator"/>
    <addaction name="actionArea"/>
    <addaction name="actionContour"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    
   <widget class="QMenu" name="menuRotate">
    <property name="title">
     <string>Rotate</string>
    </property>
    <addaction name="actionRotate_Left"/>
    <addaction name="actionRotate_Right"/>
   </widget>
   <widget class="QMenu" name="menuFlip">
    <property name="title">
     <string>Flip</string>
    </property>
    <addaction name="actionFlip_Horizontal"/>
    <addaction name="actionFlip_Vertical"/>
   </widget>
    
    <addaction name="menuRotate"/>
    <addaction name="menuFlip"/>
    
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionTab_Out"/>
    <addaction name="separator"/>
    <addaction name="actionMode_Arrangement"/>
    <addaction name="actionMode_CellComplex"/>
    <addaction name="actionMode_Clearance"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuGenerate">
    <property name="title">
     <string>Generate</string>
    </property>
    <addaction name="actionExtrude"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGenerate"/>
  </widget>
  
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockToolbox">
   <property name="windowTitle">
    <string>Toolbox</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="BlueprintToolbox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>182</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Default</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockProperties">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionArea">
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="actionContour">
   <property name="text">
    <string>Contour</string>
   </property>
  </action>
  <action name="actionExtrude">
   <property name="text">
    <string>Extrude</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionToolbox">
   <property name="text">
    <string>Toolbox</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionTab_Out">
   <property name="text">
    <string>Tab Out</string>
   </property>
  </action>
  
  <action name="actionMode_Arrangement">
   <property name="text">
    <string>Arrangement</string>
   </property>
  </action>
  <action name="actionMode_CellComplex">
   <property name="text">
    <string>Cell Complex</string>
   </property>
  </action>
  <action name="actionMode_Clearance">
   <property name="text">
    <string>Clearance</string>
   </property>
  </action>
  
  
  <action name="actionRotate_Left">
   <property name="text">
    <string>Rotate Left</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="text">
    <string>Rotate Right</string>
   </property>
  </action>
  <action name="actionFlip_Horizontal">
   <property name="text">
    <string>Flip Horizontally</string>
   </property>
  </action>
  <action name="actionFlip_Vertical">
   <property name="text">
    <string>Flip Vertically</string>
   </property>
  </action>
  
    
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BlueprintView</class>
   <extends>QGraphicsView</extends>
   <header>blueprintview.h</header>
  </customwidget>
  <customwidget>
   <class>BlueprintToolbox</class>
   <extends>QToolBox</extends>
   <header>blueprinttoolbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
